Sources/MapboxDirectionsCLI/main.swift

```diff
@@ -27,16 +26,34 @@ struct ProcessingOptions: ParsableArguments {
 }
 
 struct Command: ParsableCommand {
+    static var credentials: Credentials {
+        get throws {
+            guard let accessToken = ProcessInfo.processInfo.environment["MAPBOX_ACCESS_TOKEN"] ??
+                    UserDefaults.standard.string(forKey: "MBXAccessToken") else {
+                throw ValidationError("A Mapbox access token is required. Go to <https://account.mapbox.com/access-tokens/>, then set the MAPBOX_ACCESS_TOKEN environment variable to your access token.")
+            }
+            
+            let hostURL: URL?
+            if let host = ProcessInfo.processInfo.environment["MAPBOX_HOST"] ??
```

Comment:
The name of this environment variable is consistent with the [Mapbox Python SDK](https://github.com/mapbox/mapbox-sdk-py/blob/e56127b96eea150831c84b67a779378598790fd4/mapbox/services/base.py#L85), which similarly reads credentials from the environment. It differs from the [`MGLMapboxAPIBaseURL`](https://github.com/mapbox/mapbox-directions-swift/blob/2db7fda6029a7c9987cfb9fc7975061891820ca3/Sources/MapboxDirections/Credentials.swift#L8) Info.plist key that applications would set, but the naming conventions for environment variables are different than for Info.plist keys anyways.